### ISSUE TITLE
dev version may work better than last 18 month old official version, on current stack

### ISSUE THREAD

**Issue by stonebig** [jakevdp/mpld3#311]
Reading this: https://github.com/ioam/holoviews/issues/230#issuecomment-131214342

Would it be possible to get an intermediate 0.2.1  ?

**Comment by danielballan** [comment]
That seems reasonable. I'd be willing to do the github/PyPI wrangling if no one else has the bandwidth. Thoughts, @jakevdp? We'd have to at least handle #303. Is anything else critical?

**Comment by philippjfr** [comment]
This would be great. I've been hoping for a v0.3 release for a while now, but following the issue on that topic I see that there are various things you still want to address. In terms of HoloViews, we only really rely on a few things such as the NumpyEncoder, which I could easily duplicate as it's fairly short. However, I do think there's plenty of new features, including various plugins, that deserve a release sooner rather than later, so we and presumably many other people can build on them. Maybe I'll even have a few minutes to look at #303 myself.

**Comment by jakevdp** [comment]
Yep – I was getting ready to do an 0.3, but got held up by a bug that was introduced. I think we should be able to do it now

**Comment by danielballan** [comment]
@jakevdp Was the v0.2.0 tag never pushed to GitHub? Only v0.1.0 shows on our GitHub "releases" page, and git doesn't see v0.2.0 either.

```
$ git fetch upstream --tags
$ git tag
v0.0.1
```

**Comment by danielballan** [comment]
Update on v0.3.0 deployment progress:
* A bug report on mplexporter from August 2014, mpld3/mplexporter#29, is still open; two mplexporter tests are still failing on mpl 1.4.x. Roll with it?
* The JS tests are failing badly for me. I suspect a local problem (or an issue between chair and keyboard). Could someone run the JS tests on master and let me know how it goes?

**Comment by jakevdp** [comment]
Hmm – I think I may have forgotten to push the tag. We do have an 0.2.x branch – I just generated the tag and pushed it.

**Comment by jakevdp** [comment]
> two mplexporter tests are still failing on mpl 1.4.x

This is a huge issue IMO. I think that many people will have moved from mpl 1.3 by now... this brings up one of the huge challenges of mpld3: it is fundamentally a hack that relies on private methods and structures in matplotlib which can and will change without warning from release to release.

I suppose we can just roll with it, and add these to the (very long) known failures list... Fixing Python 3 support in the release seems to be important to people.

**Comment by jakevdp** [comment]
JS tests all pass for me. I wonder if its something different in our npm installations?

**Comment by danielballan** [comment]
> This is a huge issue IMO.

I agree. In fact, we should be testing against 1.5.0rc2. I expect the Fernando-inspired labeled data API, `plt.plot('x', 'y', data=data)`, will push many people to upgrade. I spend three hours each day on the train typing next to Captain Matplotlib Internals... maybe we can get those two tests patched up if this release can wait another week.

>  I wonder if its something different in our npm installations?

``
$ npm --version
2.10.0
```

**Comment by jakevdp** [comment]
Yeah, I'm on version 1.3.7. Not that I have any intuition into whether that might be an issue :smile:

**Comment by danielballan** [comment]
A pair of veteran JS experts, we are. :- )
